spec/models/book_spec.rb

```diff
@@ -2,4 +2,6 @@
 
 RSpec.describe Book, type: :model do
   it { is_expected.to validate_presence_of(:title) }
+  it { is_expected.to have_many(:book_lists) }
+  it { is_expected.to have_many(:lists).through(:book_lists) }
 end
```

Comment:
I think it would be good to add the foreign key to the specs. That way there is some test coverage that would fail if another developer accidentally removes that migration/adds another migration that removes the foreign key.

It's seems like a good practice to have test coverage of that so that a regression would break a test and let the developer know that there could be orphaned foreign keys in the db.